tests/test_preprocessing_utils.py

```diff
@@ -536,6 +537,41 @@ def create_data(self):
                 },
             ],
         ]
+        self.target_metadata_entity_paragraph = [
+            [],
+            [
+                {
+                    "char_end_idx": 120,
+                    "char_start_idx": 0,
+                    "key": "entity_paragraph",
+                    "relative_end_pos": 0,
+                    "relative_start_pos": 0,
+                    "type": "local",
+                    "value": "Barack_Obama",
+                },
+                {
+                    "char_end_idx": 120,
+                    "char_start_idx": 0,
+                    "key": "entity_paragraph",
+                    "relative_end_pos": 1,
+                    "relative_start_pos": 1,
+                    "type": "local",
+                    "value": "Angela_Merkel",
+                },
+            ],
```

Comment:
Just want to be sure I understand the intention: the two examples mean "a paragraph [0,120) containing two entities; the first one is Barack_Obama and the second is Angela_Merkel." Is it correct?

Reply:
Yes, you are correct!